### PR TITLE
Remove the mPulse beacon from our pages

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #NUMBER

**Overall change:**
now that our trial with mPulse has come to a close we should remove the mPulse beacon from Simorgh.

This PR removes the mPulse beacons container as well as the relevant CSP values.

**Code changes:**

- Remove mPulse container
- Remove mPulse CSP values

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by FK78 — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -8416,7 +8416,7 @@
     },
     "diffable-html": {
       "version": "4.0.0",
-      "resolved": "https://registry.npmjs.org/diffable-html/-/diffable-html-4.0.0.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/diffable-html/-/diffable-html-4.0.0.tgz",
```
Comment (j-pendlebury) [inline package-lock.json]: this change should probably be removed

Comment (chris-hinds) [reply package-lock.json]: yup, not sure why this is happening still :(

Comment (chris-hinds) [reply package-lock.json]: have pushed the fix @jblaut

**Review by j-pendlebury — APPROVED** [review]
Looks good 👍 bye-bye mPulse 👋

**Comment by j-pendlebury** [comment]
looks like you have an issue with a license:
```
Problems with the licenses for these dependencies:
  tslib@1.13.0
    License:     0BSD
    Repository:  https://github.com/Microsoft/tslib
    Publisher:   Microsoft Corp.
    Url:         undefined
```

Reverting this back should do the trick https://github.com/bbc/simorgh/pull/6577/files#diff-32607347f8126e6534ebc7ebaec4853dR24221-R24224

**Pull request closed without merging**